src/app/utils/item-utils.ts

```diff
@@ -296,6 +301,36 @@ export function getItemYear(item: DimItem) {
   }
 }
 
+/**
+ * Powerful Friends: 1484685887,
+ * Charge Harvester: 2263321587,
+ * Radiant Light: 2979815167
+ */
+export function isPlugStatActive(item: DimItem, plugHash: number, statHash: number) {
```

Comment:
I was going to pull this out into d2ai but as we are only handling 3 mods at the moment I didn't bother. Would be happy to if people prefer though.

Reply:
We need to handle masterwork mods as well - their conditional stats are only active on adept weapons.

Reply:
Ah ok, I hadn't considered adept weapons so I will figure that out.